tests/index.js

```diff
@@ -85,6 +85,7 @@ require('./bugs/issue267.js');
 require('./bugs/issue270.js');
 require('./bugs/rename-dir-trailing-slash.js');
 require('./bugs/issue357.js');
+require('./bugs/issue681.js');
```

Comment:
You don't need to add a new file for this bug.  You're updating the code in `tests/spec/filer.filesystem.spec.js`, which already exists, and you've addressed below.  You can remove this and `tests/bugs/issue681.js`:

```
git rm tests/bugs/issue681.js
git checkout master tests/index.js
```

This will undo your changes.

Reply:
Removed file and require line